vpc.tf

```diff
@@ -84,6 +84,32 @@ resource "aws_security_group" "vpc_public_sg" {
   }
 }
 
+resource "aws_security_group" "vpc_memcached_sg" {
```

Comment:
this name need to be more generic so that we can use it for all sorts of private resources